changelog.d/issue-7370

```diff
@@ -0,0 +1,3 @@
+synopsis: Documented limitation to compiler section of cabal project files
+issues:
+  #7370
```

Comment:
No need for a changelog entry in this case, it isn't an user facing change

Reply:
Do I just re-push without the file?

Reply:
Yes